app/build.gradle

```diff
@@ -48,6 +48,7 @@ dependencies {
     implementation libraries.androidxCoordinatorLayout
     implementation libraries.androidxCoreKtx
     implementation libraries.androidxMaterial
+    implementation libraries.androidxPagingRuntime
```

Comment:
I guess this includes paging common. Since we don't have a clean arch approach (where we want to avoid android libraries in kotlin-only modules), we can just stick this dependency in and get everything.